apps/src/storage/dataBrowser/dataVisualizer/VisualizerModal.jsx

```diff
@@ -153,13 +158,20 @@ class VisualizerModal extends React.Component {
             </div>
 
             <DropdownField
-              displayName="Chart Type"
-              options={['Bar Chart', 'Histogram', 'Scatter Plot', 'Cross Tab']}
+              displayName={msg.dataVisualizerChartType()}
+              options={[
+                ChartType.BAR_CHART,
+                ChartType.HISTOGRAM,
+                ChartType.SCATTER_PLOT,
+                ChartType.CROSS_TAB
+              ]}
               value={this.state.chartType}
-              onChange={event => this.setState({chartType: event.target.value})}
+              onChange={event =>
+                this.setState({chartType: parseFloat(event.target.value)})
+              }
             />
 
-            {this.state.chartType === 'Histogram' && (
+            {this.state.chartType === ChartType.HISTOGRAM && (
               <div style={styles.input}>
                 <label style={rowStyle.description}>Bucket Size</label>
```

Comment:
Does this one need to i18n too?

Reply:
🤦‍♀ Yes, my bad. Will do in a quick follow up.